rosdep/python.yaml

```diff
@@ -2811,6 +2811,39 @@ python-tk:
     wily_python3: [python3-tk]
     xenial: [python-tk]
     xenial_python3: [python3-tk]
+python-torch-cuda75-pip:
+  debian:
+    pip:
+      packages: ['http://download.pytorch.org/whl/cu75/torch-0.1.11.post4-cp27-none-linux_x86_64.whl']
```

Comment:
`-U` means update if there is an update available. It shouldn't reinstall if it's already at the latest version. From the help.

```
-U, --upgrade               Upgrade all specified packages to the newest available version. This process is recursive regardless of whether a dependency
                              is already satisfied.

```

If it's reinstalling every time that suggests that it's not detecting it correctly. I think the problem is that using the URL it doesn't know the version requested until after it's already downloaded it.